src/components/Content.tsx

```diff
@@ -114,6 +114,7 @@ const Content: ContentComponent = props => {
         ? <Search />
         : <React.Fragment>
           {rootThoughtsLength === 0 ? <NewThoughtInstructions childrenLength={rootThoughtsLength} isTutorial={isTutorialLocal} /> : <Subthoughts
+            thoughtsResolved={[]}
```

Comment:
I might suggest making `path` optional. If it is not present we can simply use `simplePath`.